service/github/github.go

```diff
@@ -323,12 +323,22 @@ func (g *PullRequest) diffUsingGitCommand(ctx context.Context) ([]byte, error) {
 
 	headSha := pr.GetHead().GetSHA()
 
-	mergeBaseBytes, err := exec.Command("git", "merge-base", headSha, pr.GetBase().GetSHA()).Output()
+	commits, _, err := g.cli.Repositories.CompareCommits(ctx, g.owner, g.repo, headSha, pr.GetBase().GetSHA(), nil)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get merge-base commit: %w", err)
+		return nil, err
+	}
+	if commits == nil || commits.MergeBaseCommit == nil || commits.MergeBaseCommit.SHA == nil {
+		return nil, errors.New("cannot to get merge base commit")
 	}
 
-	mergeBase := strings.Trim(string(mergeBaseBytes), "\n")
+	mergeBase := *commits.MergeBaseCommit.SHA
+
+	if os.Getenv("SKIP_GIT_FETCH") != "true" {
```

Comment:
Please add REVIEWDOG_ prefix

Reply:
I add it: https://github.com/reviewdog/reviewdog/pull/1714/commits/ef263f7abf78f18487cfce34a57746dc63f1e7a9